tutorials/axios/axios-all.md

```diff
@@ -0,0 +1,5 @@
+`axios.all()` is axios's own way of making concurrent http requests and getting back an array of the responses. Because you would be making multiple requests when using this, you should make an effort to make the code more readable. Making a request would look something like as follows:
```

Comment:
The "Because you would be making multiple requests when using this, you should make an effort to make the code more readable" sentence is a little awkward. Maybe just "We recommend assigning the return value of `axios.get()` to a variable before passing it into `axios.all()` to make your code more readable."?